src/main/java/space/obminyashka/items_exchange/service/impl/SendGridService.java

```diff
@@ -84,7 +83,18 @@ private static Personalization createPersonalizationAndSetParameters(EmailType e
             personalization.addDynamicTemplateData(key, getMessageSource(parameterSource));
         });
 
-        personalization.addDynamicTemplateData("url", host.concat(emailType.callbackEndpoint.replace("{code}", codeId.toString())));
+        String htmlAction = """
+                <p style="width: 420px; height: 84px; margin: 8px auto 0 auto; font-size: 18px">
+                    %s 
+                    <a href="https://obminyashka.space">obminyashka.space</a>! 
```

Comment:
That prevents local use of the feature.
It'd be great to pass here incoming **host** value